datalad_container/containers_add.py

```diff
@@ -199,6 +199,16 @@ def __call__(name, url=None, dataset=None, call_fmt=None, image=None,
                              purpose='add container')
         runner = WitlessRunner()
 
+        if image is not None:
+            if op.isabs(image):
```

Comment:
Docstring does say relative so I guess it is good. I just wanted to raise a comment that on general we are ok with providing absolute paths everywhere and I think we (or git) normalize them into relative if needed... so it would've been more consistent to just normalize absolute path to correct relative here